app/models/types/adaptations.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class Adaptations(str, Enum):
```

Comment:
Can you add a comment explaining that this list comes from the CLA CHS.

Reply:
Added